app/models/character.rb

```diff
@@ -19,6 +19,7 @@ class Character < ApplicationRecord
 
   has_many :character_tags, inverse_of: :character, dependent: :destroy
   has_many :settings, -> { ordered_by_char_tag }, through: :character_tags, source: :setting, dependent: :destroy
+  has_many :unordered_settings, through: :character_tags, source: :setting, dependent: :destroy
```

Comment:
I wouldn't expect that to remove any queries, given it's going through char_tags anyway? unless rails magic will join those without the intervening model.

Reply:
So the problem here was that because of the `-> { ordered_by_char_tag }`, I couldn't use the existing :settings relation in an .includes. So adding a second relation like this does enable removing queries.